scalameta/semanticdb-scalac-core/src/main/scala/scala/meta/internal/semanticdb/DenotationOps.scala

```diff
@@ -105,9 +105,17 @@ trait DenotationOps { self: DatabaseOps =>
       }
     }
 
+    private def overrides: List[m.Symbol] =
+      if (config.overrides.isAll) gsym.overrides.map(_.toSemantic)
+      else Nil
+
     def toDenotation: m.Denotation = {
       val (minfo, mnames) = info
-      m.Denotation(flags, name, minfo, mnames)
+      m.Denotation(flags, name, minfo, mnames, Nil, overrides)
     }
+
+    def overridesMembers: List[(m.Symbol, m.Denotation)] =
+      if (config.overrides.isAll) gsym.overrides.map(ov => (ov.toSemantic, ov.toDenotation))
```

Comment:
This circumvents the `config.denotations` setting. Here's how I propose we take it into account:

```
if (config.overrides.isAll) {
  gsym.overrides.flatMap { ov => 
    if (!config.denotations.saveReferences && ov.symbol.pos.isEmpty) None
    else Some((ov.toSemantic, ov.toDenotation))
  }
}
```